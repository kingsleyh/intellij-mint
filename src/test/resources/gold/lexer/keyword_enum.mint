enum Fruit {
  Apple
  Pear
  Banana
}