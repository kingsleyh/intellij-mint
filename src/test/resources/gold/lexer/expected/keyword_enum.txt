enum ('enum')
WHITE_SPACE (' ')
CAPITALISED_IDENT ('Fruit')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n  ')
CAPITALISED_IDENT ('Apple')
WHITE_SPACE ('\n  ')
CAPITALISED_IDENT ('Pear')
WHITE_SPACE ('\n  ')
CAPITALISED_IDENT ('Banana')
WHITE_SPACE ('\n')
} ('}')